pyaedt/application/Analysis.py

```diff
@@ -1695,6 +1704,8 @@ def analyze_setup(self, name, num_cores=None, num_tasks=None, num_gpu=None, acf_
             if self.design_type == "Icepak":
                 use_auto_settings = False
             update_hpc_option(target_name, "UseAutoSettings", use_auto_settings, False)
+            if num_variations_to_distribute:
+                update_hpc_option(target_name, 'NumVariationsToDistribute', num_variations_to_distribute, False)
```

Comment:
@nikolarobottesla This should fix the failing action named "style".
```suggestion
                update_hpc_option(target_name, "NumVariationsToDistribute", num_variations_to_distribute, False)
```

Reply:
@nikolarobottesla Thanks again for contributing to the project.